Std/Tactic/RunCmd.lean

```diff
@@ -32,6 +32,17 @@ elab (name := runCmd) "run_cmd " elems:doSeq : command => do
       (mkApp (mkConst ``CommandElabM) (mkConst ``Unit))
       (← `(discard do $elems))
 
+/--
+The `run_meta doSeq` command executes code in `MetaM Unit`.
+This is almost the same as `#eval show MetaM Unit from do doSeq`,
+except that it doesn't print an empty diagnostic.
+-/
+elab (name := runMeta) "run_meta " elems:doSeq : command => do
```

Comment:
This is misnamed and mis-documented now since `$elems` is expecting a TermElabM.

Reply:
With your proposed change I think this is back to being correct?

Reply:
assuming I got the syntax correct...